requirements/edx/base.txt

```diff
@@ -119,7 +119,7 @@ django_debug_toolbar==1.3.2
 
 # Used for testing
 astroid==1.3.8
-bok-choy==0.4.6
+#bok-choy==0.4.6
```

Comment:
Yea, I'll update the version once that has landed.  This PR will probably change quite a bit though.  I've pulled the custom rules out into a [separate repository](https://github.com/clytwynec/edx-custom-a11y-rules) (still needs to be be made an actual edx repo and have a formal review as well)  so that we will be able to include them in other places (edx-analytics-repo, ora-2, etc.).